packages/extension-ui/src/Popup/Signing/Signing.test.tsx

```diff
@@ -182,46 +228,70 @@ describe.skip('Signing requests', () => {
 
   describe('Request rendering', () => {
     it('correctly displays request 1', () => {
-      expect(wrapper.find(Address).find('FullAddress').text()).toBe('5D4bqjQRPgdMBK8bNvhX4tSuCtSGZS7rZjD5XH5SoKcFeKn5');
+      expect(wrapper.find(Address).find('.fullAddress').text()).toBe(signRequests[0].account.address);
       expect(wrapper.find(Extrinsic).find('td.data').map((el): string => el.text())).toEqual([
-        'polkadot.js',
-        'Alexander',
-        '112',
-        '0',
-        'balances.transfer',
-        `{
-  "dest": "5D4bqjQRPgdMBK8bNvhX4tSuCtSGZS7rZjD5XH5SoKcFeKn5",
-  "value": 31415926500000
-}`,
-        ' Transfer some liquid free balance to another account.   `transfer` will set the `FreeBalance` of the sender and receiver.  It will decrease the total issuance of the system by the `TransferFee`.  If the sender\'s account is below the existential deposit as a result  of the transfer, the account will be reaped.   The dispatch origin for this call must be `Signed` by the transactor.',
-        'mortal, valid from #4,033,610 to #4,033,674'
+        'https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwestend-rpc.polkadot.io#/accounts',
+        '0xe143f23803ac50e8f6f8e62695d1ce9e4e1d68aa36c1cd2cfd15340213f3423e',
+        '45',
+        '3',
+        '0x0403c6111b239376e5e8b983dc2d2459cbb6caed64cc1d21723973d061ae0861ef690b00b04e2bde6f',
+        'mortal, valid from {{birth}} to {{death}}'
       ]);
     });
 
     it('correctly displays request 2', () => {
-      wrapper.find('ArrowRight').simulate('click');
-      expect(wrapper.find(Address).find('FullAddress').text()).toBe('5E9nq1yGJJFiP8C75ryD9J2R62q2cesz6NumLnuXRgmuN5DG');
-      expect(wrapper.find(Extrinsic).find('td.data').at(5).text()).toBe(`{
-  "dest": "5D1ss3KFnzNtLzRDfUhqLivzVvt5BDrBnK21dMf1si2twPuj",
-  "value": 10000000000
-}`);
+      wrapper.find('FontAwesomeIcon.arrowRight').simulate('click');
+      expect(wrapper.find(Address).find('.fullAddress').text()).toBe('5E9nq1yGJJFiP8C75ryD9J2R62q2cesz6NumLnuXRgmuN5DG');
+      expect(wrapper.find(Extrinsic).find('td.data').map((el): string => el.text())).toEqual([
+        'https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwestend-rpc.polkadot.io#/accounts',
+        '0xe143f23803ac50e8f6f8e62695d1ce9e4e1d68aa36c1cd2cfd15340213f3423e',
+        '45',
+        '3',
+        '0x0403c6111b239376e5e8b983dc2d2459cbb6caed64cc1d21723973d061ae0861ef690b00b04e2bde6f',
```

Comment:
I might need a tip here @jacogr. This extrinsic (a transfer) isn't decoded in the test. Do you know a way for me load the latest metadata?